### DISCUSSION TITLE
For changes in a subproject's workflowitem ordering, the history doesn't show _what_ has changed

### DISCUSSION THREAD

**Discussion by phpai** [openkfw/TruBudget#866] [Ideas]
In GitLab by @kevinbader on Aug 14, 2018, 11:25

From #47:

Currently the history item when the order of the workflowitems change looks like: 
```text
mstein changed the workflowitem ordering 
```

It would be nice to get details about what exactly changed.

**Comment by mathiashoeld** [comment]
relates to #236
